_data/strings.yml

```
@@ -14,6 +14,14 @@ locales:
     made_possible: "Dieses Projekt wird von den <a href='https://github.com/qmk'>QMK-Entwicklern</a> und Mitwirkenden wie Ihnen unterstützt!"
     hosted_on: "Betrieben mit GitHub Pages &mdash; Layout von <a href='https://github.com/orderedlist'>orderedlist</a>"
     description: "Eine Open Source Firmware für auf AVR oder ARM basierende Tastaturen"
+  pt-br: &DEFAULT_PT_BR
+    <<: *DEFAULT_EN
+    language_name: "Português"
+    language: "Linguagem"
+    qmk_firmware: "Firmware QMK"
+    made_possible: "Este projeto é mantido por <a href='https://github.com/qmk'>colaboradores do QMK</a> e colaboradores como você!"
+    hosted_on: "Hospedado em páginas &mdash do GitHub; Tema por <a href='https://github.com/orderedlist'>lista ordenada</a>"
```

GitHub Pages (in English) is a product name, and `orderedlist` is a GitHub username. The `&mdash;` is used to separate these bits of information visually.

I think the below would be better.

```suggestion
    hosted_on: "Hospedado em Páginas do GitHub &mdash; Tema por <a href='https://github.com/orderedlist'>orderedlist</a>"
```